### ISSUE TITLE
Formula Lookup needs documentation

### ISSUE THREAD

**Issue by sbauch** [Airtable/airtable-ruby#11]
So I wrote [this functionality](https://github.com/Airtable/airtable-ruby/issues/4) and I have no idea how to use it.

I've been trying to use https://support.airtable.com/hc/en-us/articles/203255215-Formula-Field-Reference but it really isn't all that helpful.

For instance, I wanted to select only the records where a boolean column is true. For the formula string supplied to `select` I tried

`approved = true`
`Approved = true`
`Approved IS true`
`TRUE(Approved)`

until I landed, with a wild guess, on `Approved = 1` which worked.

I'm happy to *help* document this, but I feel like I don't have enough knowledge about Airtable's code base to really do it accurately.

**Comment by syrnick** [comment]
Fair point. We use Javascript semantics for truthfulness, so 0, null, [], '' would all be falsey, 1, 'foo', [0]. That should cover the basic expression

For complex formulas, I'd recommend setting up the formula in the UI first to validate what you're getting. You can wrap any condition you're not sure about with `IF(condition, 1, 0)`
